.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: Run tests
+on: [push, pull_request]
+
+jobs:
+  run:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node: [8, 10, 12]
+
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 3
+
+    - name: Set Node.js version
+      uses: actions/setup-node@v1.1.0
+      with:
+        version: ${{ matrix.node }}
+
+    - run: node --version
+    - run: npm --version
+    - run: java -version
```

Comment:
Where do we need Java ? 😯

Reply:
1, Java is already installed anyway
2. It's needed for vnu-jar